cmd/kpod/ps.go

```diff
@@ -135,12 +135,16 @@ func psCmd(c *cli.Context) error {
 	if c.IsSet("format") {
 		format = c.String("format")
 	}
+	last := -1
+	if c.IsSet("last") {
```

Comment:
Should we convert the other params to do this kind of check/cet too?

Reply:
Not for this commit, but at some point.

Reply:
Why not set the default value to -1, then you don' need any of this code.